### PR TITLE
Make aliases work on windows

### PR DESCRIPTION
**Pull request by Mikkel-T**
## Changes

- Change all occurrences of path.posix to path.

### CODE REVIEW

File: packages/astro/src/vite-plugin-config-alias/index.ts [hunk 1]
```diff
@@ -73,6 +73,7 @@ const getConfigAlias = (cwd: string | undefined): Alias[] | null => {
 export default function configAliasVitePlugin(astroConfig: { projectRoot?: URL; [key: string]: unknown }): vite.PluginOption {
   /** Aliases from the tsconfig.json or jsconfig.json configuration. */
   const configAlias = getConfigAlias(astroConfig.projectRoot && url.fileURLToPath(astroConfig.projectRoot));
+  console.log(configAlias)
```
Comment (matthewp) [inline packages/astro/src/vite-plugin-config-alias/index.ts:76]: Left in debugging, I suspect

**Comment by matthewp** [comment]
Just need to remove the console.log, otherwise looks great!

**Comment by Mikkel-T** [comment]
😅 Oops!

**Review by jonathantneal — APPROVED** [review]

**Comment by jonathantneal** [comment]
Hey all, @Mikkel-T and I paired on this over Discord and get the fix worked out. Thanks, @Mikkel-T!

**Pull request merged**